src/main/java/com/example/bookhub/controller/OrderController.java

```diff
@@ -0,0 +1,73 @@
+package com.example.bookhub.controller;
+
+import com.example.bookhub.dto.order.CreateOrderRequestDto;
+import com.example.bookhub.dto.order.OrderItemWithoutPriceResponseDto;
+import com.example.bookhub.dto.order.OrderResponseDto;
+import com.example.bookhub.dto.order.UpdateOrderStatusRequestDto;
+import com.example.bookhub.service.OrderService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.Valid;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PatchMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Order Management", description = "Endpoints for managing orders")
+@RestController
+@RequestMapping("/api/orders")

```

Comment:
add `server.servlet.context-path=/api` to `application.properties`
```suggestion
@RequestMapping("/orders")
```